src/cljs/imcljs/internal/io.cljs

```diff
@@ -40,6 +41,16 @@
             (wrap-post-defaults options model) ; Add form params
             (wrap-auth token))))
 
+(defn put-wrapper-
+  "Returns the results of queries as table rows."
```

Comment:
Is this true, or possibly copypasta? 😆

Reply:
The same docstring is present on all the wrappers. I don't know what's real anymore!